Gemfile

```diff
@@ -365,3 +352,11 @@ gem 'mailgun-ruby', '~>1.2.14'
 gem 'mailjet', '~> 1.7.3'
 
 gem "json-schema", "~> 4.3"
+
+gem "csv"
+
+gem 'mutex_m' # needed for httpclient in Ruby >= 3.4, drop explicit dep if we upgrade httpclient
+
+gem 'abbrev' # needed for activesupport in Ruby >= 3.4, drop explicit dep after we upgrade to activesupport >= 7.2
+gem 'drb' # needed for activesupport in Ruby >= 3.4, drop explicit after we upgrade to activesupport >= 7.2
+gem 'observer' # needed for activesupport in Ruby >= 3.4, drop explicit after we upgrade to activesupport >= 7.2
```

Comment:
Ruby 3.4 (future) WILL be removing lots of built-in libraries and turn them into Gems. As a result, Ruby 3.3.4 makes lots of annoying warning noises if you have implicit deps on them. These fix those warnings.